app/layout.js

```diff
@@ -0,0 +1,12 @@
+export const metadata = {
+  title: 'Next.js',
```

Comment:
Update Title to Learning NextJS

Reply:
Fixed in [e014d4c](https://github.com/vuongwphamj/online-store-front-end/pull/5/commits/e014d4c7ad287f30709784019a8efbff1a60d798)